src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -1339,6 +1375,7 @@ class PersonAgent(
         case Some(activity) =>
           val (tick, triggerId) = releaseTickAndTriggerId()
           val activityEndTime = calculateActivityEndTime(activity, tick)
+          activity.setStartTime(tick.toDouble)
```

Comment:
Yeah good point. This was a little bit tricky because I think it only comes up during the ActivitiesDurationSpec, where it's used by beam.agentsim.agents.PersonAgent#calculateActivityEndTime, but I'll also set end time here for consistency and confirm that it doesn't lead to any problems